### PR TITLE
Pull request-multiply

### PR DESCRIPTION
**Pull request by SalehPouresmaeeli**

### CODE REVIEW

**Review by t-rooker — APPROVED** [review]
looks good!

**Review by aplgit — APPROVED** [review]
Amazing! I think that's perfect....maybe!!! :-D

File: pythoncalculator/multiply.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (tim-rook) [inline pythoncalculator/multiply.py]:
```suggestion
    return x * y
```

**Review by t-rooker — CHANGES_REQUESTED** [review]
Also the pythoncalculator/add.py and tests/tests_add.py are missing, need to be reinstated

**Review by aplgit — APPROVED** [review]

**Review by t-rooker — APPROVED** [review]

**Pull request merged**